server/hubot.coffee

```diff
@@ -138,7 +141,6 @@ class BlackboardAdapter extends Hubot.Adapter
 IGNORED_NICKS =
   'codexbot': true
   '': true
-  'thehunt': true
```

Comment:
Not intentional?

Reply:
I removed it after I added the bot_ignore field, since everything from mailwatch is ignored anyways.